### ISSUE TITLE
PlatformCompatibilityAnalyzer should report a warning for `UnsupportedOSPlatform` attribute in interface implementation

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#77256]
If an implementation of an interface member has `UnsupportedOSPlatform`, while the interface member itself doesn't, this should be a warning. For example:

```csharp
public interface IMyInterface
{
    public string MyProperty { get; }
}

public class MyImpl : IMyInterface
{
    [UnsupportedOSPlatform("browser")]
    public string MyProperty
    {
        get
        {
            // ...
        }
    }
}
```

Because, MyProperty can still be accessed via the interface without a warning, masking an issue.

**Comment by Youssef1313** [comment]
Wrong repo, sorry.
